docs/CHILI-GraFx/users/scope/index.md

```diff
@@ -1,26 +1,30 @@
 # User management
 
-## Scope of permissions
+## Scope of roles
 
-The scope of the permissions is the concept that not all permissions have the same impact. Some permissions have impact on a different level.
```

Comment:
@BramVerniest I saw you replaced 'permissions' by 'access', so maybe you don't want to use 'permissions' here?